src/components/TextField/TextField.jsx

```diff
@@ -132,9 +134,12 @@ const TextField = forwardRef(
               onBlur={onBlur}
               onFocus={onFocus}
               onKeyDown={onKeyDown}
-              aria-label={inputAriaLabel || placeholder}
               maxLength={maxLength}
+              role={searchResultsContainerId && "combobox"}
```

Comment:
what is this "combobox"?

Reply:
A composite widget containing a single-line textbox and another element such as listbox or grid, that can dynamically pop up to help the user set the value of the textbox.

It used here to describe the voice reader, that even thought I am focused on the search, I have an extended part of results I want you to read. This container Id is given in the aria-owns.
And the active result is given in the aria-activedescenedent :)